coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 58.27
```

Comment:
delete file, we haven't been  pushing the coverage stuff

Reply:
removed